sktime/transformations/series/theta.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3 -u
+# -*- coding: utf-8 -*-
+"""copyright: sktime developers, BSD-3-Clause License (see LICENSE file)."""
+
+__author__ = ["Markus Löning"]
+# __all__ = ["ThetaLinesTransformer"]
+
+import numpy as np
+import pandas as pd
+
+from sktime.transformations.base import _SeriesToSeriesTransformer
+from sktime.forecasting.base import ForecastingHorizon
+from sktime.forecasting.trend import PolynomialTrendForecaster
+
+
+class ThetaLines(_SeriesToSeriesTransformer):
+    """Decompose the original data into two or more ThetaLines."""
```

Comment:
Maybe include reference to footnote `[1]` here, too?

Reply:
I don't know if sktime has 'rules' for how to reference.  Here is example where reference is in the docstring of the class: https://github.com/alan-turing-institute/sktime/blob/main/sktime/forecasting/theta.py

@mloning. For a references to papers, is there a norm for it to be in the docstirng of the class, or docstring of a method within the class, or ... ? Or does it basically not really matter as long as it is somewhere